Management_System/Management_System/Entities/Task.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Management_System.Enums;
+using System.Xml.Linq;
+
+namespace Management_System.Entities
+{
+    public class Task
```

Comment:
I was thinking about name of this class. We already have class `Task` in the .Net.